lib/dirList.js

```diff
@@ -159,10 +160,7 @@ const dirList = {
    * @return {boolean}
    */
   handle: function (route, options) {
-    if (!options.names) {
-      return false
-    }
-    return options.names.includes(path.basename(route)) ||
+    return options.names?.includes(path.basename(route)) ||
```

Comment:
I always prefer optional chaining but just wanna point out that these are not equivalent as `undefined` will be returned instead of `false` if `options.names` does not exist, and this function returns a boolean according to its type